imapserver/capability.go

```diff
@@ -81,6 +81,7 @@ func (c *Conn) availableCaps() []imap.Cap {
 			imap.CapCreateSpecialUse,
 			imap.CapLiteralPlus,
 			imap.CapUnauthenticate,
+			imap.CapChildren,
```

Comment:
In #693, CHILDREN was in the list of caps that only make sense with IMAP4rev1, and that don't make sense in IMAP4rev2. This was correct AFAIU, because IMAP4rev2 also defines these attributes and additionally a CHILDREN list return option.

Reply:
ok, added to rev1 only and folded in rev2 caps.